src/integrators/aov.cpp

```diff
@@ -199,6 +207,19 @@ class AOVIntegrator final : public SamplingIntegrator<Float, Spectrum> {
 
         for (size_t i = 0; i < m_aov_types.size(); ++i) {
             switch (m_aov_types[i]) {
+                case Type::Albedo: {
+                    BSDFPtr bsdf = si.bsdf(ray);
+                    Spectrum spec = bsdf->get_diffuse_reflectance(si, active);
+                    Color3f color;
+                    if constexpr (is_spectral_v<Spectrum>)
+                        color = spectrum_to_srgb(spec, ray.wavelengths, active);
+                    else 
+                        color = spec;
+                    *aovs++ = dr::select(si.is_valid(), color.r(), 0.f);
+                    *aovs++ = dr::select(si.is_valid(), color.g(), 0.f);
+                    *aovs++ = dr::select(si.is_valid(), color.b(), 0.f);
```

Comment:
Do we need the `dr::select()` statements here? in case `si.is_valid() = false` the `bsdf` pointer should be `NULL` hence the virtual function call `get_diffuse_reflectance()` should already return zeros for those entries.